### ISSUE TITLE
GCC: Libiberty 

### ISSUE THREAD

**Issue by stachon** [hpcugent/easybuild-easyblocks#969]
Hello,
can we add Libiberty to GCC build? (adding --enable-install-libiberty to configure) ? In versions 4.8.3 and 4.7.4 it is present, it seems that it is no longer installed by default since GCC 4.8.x and binutils-2.24. Or perhaps this should go to binutils? (it seems that either GCC, binutils and GDB can provide it).

[1] https://gcc.gnu.org/onlinedocs/libiberty/Using.html#Using

**Comment by boegel** [comment]
@stachon I see no reason not to do it...

We have an easyblock for both `binutils` and `GCC`, and since we're using our own `binutils` in recent toolchains, that seems a good place to add it?

@wpoely86 thoughts?

**Comment by wpoely86** [comment]
Fine by me but wasn't there a reason we don't build it? 

But doesn't hurt to try it again.

**Comment by ocaisa** [comment]
See https://github.com/hpcugent/easybuild-easyconfigs/pull/1885#issuecomment-133078819 and the few comments after for why we didn't include libiberty in the past

**Comment by boegel** [comment]
enabling the installation of `libiberty` (by default) is done in #1030

**Comment by boegel** [comment]
It seems like this was our reason for not enabling the installation of `libiberty` by default (error I got when building `binutils` 2.25 with `GCC` 4.9.3 that has `libiberty` included:

```
/gpfs/scratch/projects/project_gpilot/vsc40023/eb_rpath/software/binutils/2.25/bin/ld.gold: error: /gpfs/scratch/projects/project_gpilot/vsc40023/eb_rpath/software/GCCcore/4.9.3/lib64/libiberty.a(cplus-dem.o): requires unsupported dynamic reloc 11; recompile with -fPIC
```

**Comment by boegel** [comment]
#1030 (which will be included for EasyBuild v3.0) now enables the installation if `libiberty` by default through the `binutils` easyblock, so I'm considering this fixed

Thanks for reporting this @stachon!
